website/docs/r/org_vdc_nsxt_network_profile.html.markdown

```diff
@@ -0,0 +1,70 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_org_vdc_nsxt_network_profile
+sidebar_current: "docs-vcd-resource-vcd-org-vdc-nsxt-network-profile"
+description: |-
+  Provides a resource to manage NSX-T Org VDC Network Profile.
+---
+
+# vcd\_org\_vdc\_nsxt\_network\_profile
+
+Provides a resource to manage NSX-T Org VDC Network Profile.
```

Comment:
Do I understand correctly that we only get one instance of this resource for each VDC?
If yes, we should make it clear, especially because in the import instructions we reference it with Org+VDC names.

Reply:
Yes. This is a single instance per VDC. I added an explicit note